Opens a blank screen after updating

I've just updated to version 1.4. After that all I get is a blank white page when I open it. Also tried removing completely and reinstalling.

I did get the following messages during installation:

>> Local Npm module "grunt-contrib-jshint" not found. Is it installed?
>> Local Npm module "grunt-jscs" not found. Is it installed?
>> Local Npm module "grunt-lintspaces" not found. Is it installed?

Ah, it looks like bundling which worked in local development and testing within a development folder has broken in a published environment:

```bash
$ /usr/local/lib/node_modules/slack-for-linux/./webkitbuilds/slack4linux/linux64/slack4linux
[17371:0430/010941:ERROR:browser_main_loop.cc(170)] Running without the SUID sandbox! See https://code.google.com/p/chromium/wiki/LinuxSUIDSandboxDevelopment for more information on developing with the sandbox on.
[17371:0430/010941:ERROR:nw_shell.cc(335)] Error: Cannot find module 'underscore'
    at Function.Module._resolveFilename (module.js:327:15)
    at Function.Module._load (module.js:264:25)
    at Module.require (module.js:356:17)
    at require (module.js:375:17)
    at window.require (<anonymous>:4:17)
    at file:///tmp/.org.chromium.Chromium.fTGPDT/js/index.js:6:10
    at file:///tmp/.org.chromium.Chromium.fTGPDT/js/index.js:291:3
[17371:0430/010941:INFO:CONSOLE(329)] "Uncaught Error: Cannot find module 'underscore'", source: module.js (329)
[17371:0430/010941:ERROR:nw_shell.cc(335)] TypeError: Cannot read property 'load' of undefined
    at file:///tmp/.org.chromium.Chromium.fTGPDT/views/index.html:11:20
[17371:0430/010941:INFO:CONSOLE(11)] "Uncaught TypeError: Cannot read property 'load' of undefined", source: file:///tmp/.org.chromium.Chromium.fTGPDT/views/index.html (11)
[17401:0430/010952:WARNING:channel.cc(553)] Failed to send message to ack remove remote endpoint (local ID 1, remote ID 1)
```

--- a/install.sh
+++ b/install.sh
@@ -1,6 +1,10 @@
 #!/usr/bin/env sh
-# Exit upon the first error
+# Exit upon the first error and echo commands
 set -e
+set -x
+
+# Override our environment to consider all `npm` installs to be local
+npm_config_global=""
 
 # Install app's dependencies
 cd app/

--- a/run.js
+++ b/run.js
@@ -1,4 +1,6 @@
 #!/usr/bin/env node
+// Load in our dependencies
 var spawn = require('child_process').spawn;
 
-spawn(__dirname + '/./webkitbuilds/slack-for-linux/linux64/slack-for-linux');
+// Run our task and output stdout/stderr to parent
+spawn(__dirname + '/./webkitbuilds/slack-for-linux/linux64/slack-for-linux', {stdio: [0, 1, 2]});

